### ISSUE TITLE
Not sending to all the numbers automatically.Have to click back button after each send to continue further.

### ISSUE THREAD

**Issue by biswajitsahu** [nikhilmuz/WhatsApp-Bulk-Sender#17]

**Comment by nikhilmuz** [comment]
It's working fine for me. Do tell me exact circumstances to replicate it on my end or share more details to work upon.

**Comment by biswajitsahu** [comment]
Well I followed below steps.
1. Added 3 nos to the csv file.
2. Have enabled the Accessibility in settings.
3. Click on Browse button and impor the csv file.
4. Click on send.
5. It'll go to the chat window in whatsapp for the 1st number in the csv file and will send the message.
    But subsequently not sending to the 2nd no. Have to click on back button while on the 1st chat window. 
    And then it is going to the 2nd no and sending successfully. And again have to press back button to  
    send to 3rd no.

PS: I'm using the app on OnePlus 2 non rooted device running Android 6.0 .
